kitsune/products/views.py

```diff
@@ -44,12 +56,15 @@ def product_landing(request, template, slug):
         else:
             latest_version = 0
 
+    show_community_support = is_localized('get-community-support', request.LANGUAGE_CODE)
```

Comment:
I added the second query for case the article does not exist. Maybe I can change it a little and put the check for request and default locale equality outside is_localized().